math/src/msm/naive.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{
```

Comment:
This file contains the same MSM implementation we had before, just moved to this new module.